app/CMakeLists.txt

```diff
@@ -246,8 +245,12 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_VecGeom)
       ENVIRONMENT "${_env};${_geant_test_env}"
       RESOURCE_LOCK gpu
       REQUIRED_FILES "${_driver};${_gdml_inp};${_hepmc3_inp}"
-      DISABLED true
     )
+    if(NOT CELERITAS_USE_Geant4 OR NOT CELERITAS_USE_ROOT OR NOT CELERITAS_USE_HepMC3)
+      set_tests_properties("app/demo-loop" PROPERTIES
+        DISABLED false
```

Comment:
Shouldn't this be `DISABLED true`? By default the test is enabled, but if any of these dependencies is missing then we disable the test.

Reply:
:facepalm: yup